code/drasil-docLang/package.yaml

```diff
@@ -37,6 +37,7 @@ library:
   - Drasil.DocumentLanguage.Units
   - Drasil.DocumentLanguage.TraceabilityGraph
   - Drasil.SRSDocument
+  - Drasil.Sections.Requirements
```

Comment:
I _think_ that whatever we're interested in importing from `Drasil.Sections.Requirements` should instead be imported from `Drasil.SRSDocument` (I see that at the bottom of this file, there is a list of `Drasil.Sections.X` re-exports).

Reply:
The header for `Drasil.SRSDocument` says the following:

https://github.com/JacquesCarette/Drasil/blob/21ff3588595f34c3135af6e6db538e16d5318767/code/drasil-docLang/lib/Drasil/SRSDocument.hs#L1-L4

Since this function would not be common to *all* SRS examples, would it still make sense to export from here? Additionally, in `Drasil.SRSDocument` the functions from `Drasil.Sections.X` are being imported from [`Drasil.DocLang`](https://github.com/JacquesCarette/Drasil/blob/master/code/drasil-docLang/lib/Drasil/DocLang.hs).

Looking more intro `Drasil.DocLang` it exports functions from `Drasil.Sections.Requirements`, as well as the other sections:

https://github.com/JacquesCarette/Drasil/blob/21ff3588595f34c3135af6e6db538e16d5318767/code/drasil-docLang/lib/Drasil/DocLang.hs#L57-L58

What are your thoughts on exporting the function just from `Drasil.DocLang`, which is already an exposed module. Then importing from `Drasil.DocLang` wherever it is needed.

Reply:
That makes sense to me. Let's go with your suggestion :smile: